test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -1097,6 +1097,21 @@ def test_paulis_setter_rejects_bad_inputs(self):
         with self.assertRaisesRegex(ValueError, "incorrect number of operators"):
             op.paulis = PauliList([Pauli("XY"), Pauli("ZX"), Pauli("YZ")])
 
+    def test_paulis_setter_absorbs_phase(self):
```

Comment:
Could you also add a test to check the `simplify()` behavior you described in your original issue?

Reply:
Added.